### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by stevenjack**
### Problem

We want to move the `SQS` wrapper code from a number of projects into this repo.

### Solution

* Import it.

### Notes

- **Remember** to tag your merge commit after merging 👍

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+check-version:
+	git fetch && (! git rev-list ${VERSION})
+
+push-tag:
+	git checkout ${BRANCH} \
+	&& git pull origin ${BRANCH} \
+	&& git tag ${VERSION} \
+	&& git push origin ${BRANCH} --tags
+
```
Comment (revett) [inline Makefile]: 🏄 Newline

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+test_ci:
+	@docker run \
+	-v "${CURDIR}":${PATH_BASE}/${REPONAME} \
+	-w ${PATH_BASE}/${REPONAME} \
+	--entrypoint=go \
+	${GO_BUILDER_IMAGE} test ./sqs -cover
+
+
+
```
Comment (revett) [inline Makefile]: 🏄 Newlines

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+DEFAULT: test
+
+check-version:
+	git fetch && (! git rev-list ${VERSION})
+
+push-tag:
+	git checkout ${BRANCH} \
+	&& git pull origin ${BRANCH} \
+	&& git tag ${VERSION} \
+	&& git push origin ${BRANCH} --tags
```
Comment (revett) [inline Makefile]: 🏄 Get rid of these...

**Comment by stevenjack** [comment]
[[AW] Adds SQS wrapper](https://trello.com/c/MKJwOc50/882-aw-adds-sqs-wrapper)

**Pull request merged**